Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:latest
+ENV DEBIAN_FRONTEND=nonintercative
+COPY . /teuthology
+WORKDIR /teuthology
+RUN chmod +x /teuthology/bootstrap
+RUN apt-get update && apt-get install -y \ 
+    git qemu-utils python3-dev libssl-dev \
+    python3-pip python3-virtualenv vim \
+    libev-dev libvirt-dev libffi-dev \
+    libyaml-dev lsb-release && apt-get \ 
+    clean all
+RUN echo -e 'lock_server: http://paddles:8080\n\
+results_server: http://paddles:8080\n\
+queue_host: 0.0.0.0\n\
+queue_port: 11300\n\
+teuthology_path: ./' >> ~/.teuthology.yaml
```

Comment:
Might be worth storing this as a file in this directory and using `COPY` or `ADD` to include it in the image